.github/workflows/tests.yml

```diff
@@ -60,7 +59,7 @@ jobs:
             "HOST":"localhost",
             "PORT":"5432"
           }
-      run: nosetests -v --with-coverage --cover-branches --cover-package=field_audit
+      run: pynose -v --with-coverage --cover-branches --cover-package=field_audit
```

Comment:
Is this change necessary? I think `nosetests` still works with `pynose`.

Reply:
Yeah this is definitely true. I think I incorrectly viewed `nosetests` as the "deprecated" version but given that pynose exists simply to support nose as it is today, using `nosetests` sounds good. I'll undo this as the fewer changes the better.